PocketKit/Package.swift

```diff
@@ -33,11 +31,11 @@ let package = Package(
                 "Sync",
                 "Textile",
                 "Analytics",
-                "Lottie",
-                "YouTubePlayerKit",
                 "SharedPocketKit",
-                "BrazeKit",
-                .product(name: "BrazeUI", package: "BrazeKit")
+                .product(name: "YouTubePlayerKit", package: "YouTubePlayerKit"),
+                .product(name: "Lottie", package: "lottie-ios"),
+                .product(name: "BrazeKit", package: "braze-swift-sdk"),
+                .product(name: "BrazeUI", package: "braze-swift-sdk")
```

Comment:
Here's a part of SPM I still don't have knowledge in. Why the change from string literals?

Reply:
Because swift 5.7 changed it! yay!